roles/horizon/tasks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set up log rotation for horizon
+  logrotate: name=horizon path=/var/log/apache2/*.log
+  args:
+    options:
+      - daily
+      - missingok
+      - rotate 7
+      - compress
```

Comment:
These logs are already being rotated. I believe configuration comes in from the ubuntu apache package itself. I'd be curious if this task would modify the existing file or create a second file.